### PR TITLE
Bug 1150713 Remove Firefox SMS page

### PR DESCRIPTION
**Pull request by gauthierm**
Bug 1150713

Removes the SMS page, static resources and tests. Redirects the old URL to `firefox/products`

### CODE REVIEW

File: bedrock/firefox/redirects.py [hunk 1]
```diff
@@ -241,4 +241,7 @@ def firefox_mobile_faq(request, *args, **kwargs):
 
     # bug 960543
     redirect(r'^firefox/(?P<vers>[23])\.0/eula', '/legal/eula/firefox-{vers}/'),
+
+    # bug 1150713
+    redirect(r'^firefox/sms(?:/.*)?$', '/firefox/products/'),
```
Comment (pmclanahan) [inline bedrock/firefox/redirects.py]: could also just be `r'^firefox/sms/?'`

File: test_redirects/map_globalconf.py [hunk 2]
```diff
@@ -322,12 +322,14 @@
...
     url_test('/firefox/fx/', '/firefox/new/'),
 
-    # Bug 979670, 979531, 1003727, 979664, 979654, 979660
+    # Bug 979670, 979531, 1003727, 979664, 979654, 979660, 1150713
     url_test('/firefox/features/', '/firefox/desktop/'),
     url_test('/firefox/customize/', '/firefox/desktop/customize/'),
     url_test('/firefox/{performance,happy,speed,memory}/', '/firefox/desktop/fast/'),
     url_test('/firefox/security/', '/firefox/desktop/trust/'),
     url_test('/firefox/technology/', 'https://developer.mozilla.org/docs/Tools'),
+    url_test('/firefox/sms/', '/firefox/products/'),
```
Comment (pmclanahan) [inline test_redirects/map_globalconf.py]: NIT: Can (if you want) consolidate these two tests into one:

```python
url_test('/firefox/sms/{,sent}', '/firefox/products/'),
```

Not a requirement though. Just FYI.

**Comment by pmclanahan** [comment]
lgtm. I think it'll be r+ after the commits are squashed. Feel free to use or not either of my comments, they're just cosmetic.

File: bedrock/firefox/redirects.py (hunk 1 above)
Comment (gauthierm) [inline bedrock/firefox/redirects.py]: Maybe. I based the regex off one above which included the trailing `$`. If a new page with the path `sms-foo` is added the updated redirect will stomp on it.

File: test_redirects/map_globalconf.py (hunk 2 above)
Comment (gauthierm) [inline test_redirects/map_globalconf.py]: Oh, nice!

**Comment by gauthierm** [comment]
Rebased and squashed.

File: bedrock/firefox/redirects.py (hunk 1 above)
Comment (pmclanahan) [inline bedrock/firefox/redirects.py]: True. Good point.

**Pull request merged**